source/dub/compilers/buildsettings.d

```diff
@@ -149,14 +149,14 @@ struct BuildSettings {
 	// add string import files (avoids file name duplicates in addition to path duplicates)
 	private void addSI(ref string[] arr, in string[] vals)
 	{
-		outer:
+		bool[string] existing;
```

Comment:
True, I was a bit afraid to forget a place where it could get out of sync, because the arrays are public fields, so I wanted to leave that to a more in-depth optimization attempt.